demos/android/MASVS-PLATFORM/MASTG-DEMO-00de/hooks.js

```diff
@@ -0,0 +1,17 @@
+Java.perform(() => {
+  console.log('Enumerating chromium...');
+  let methods = Java.enumerateMethods('com.android.webview.chromium.*!deleteAllData')
+  console.log(JSON.stringify(methods, null, 2))
+
+  let method = methods[0]
+  let loader = method.loader;
+  let clazz = method.classes[0].name;
+
+  Java.classFactory.loader = loader;
+
+  const WebStorageAdapter = Java.use(clazz);
+  WebStorageAdapter.deleteAllData.implementation = function() {
+    console.log('WebStorage.deleteAllData() called');
+    return this.deleteAllData();
+  }
+});
```

Comment:
This is working better for me.

```suggestion
Java.perform(function () {
  function waitForMethods() {
    try {
      let methods = Java.enumerateMethods('com.android.webview.chromium.*!deleteAllData');
      if (methods && methods.length > 0) {
        console.log('Chromium methods found');
        console.log(JSON.stringify(methods, null, 2));

        let method = methods[0];
        let loader = method.loader;
        let clazz = method.classes[0].name;

        Java.classFactory.loader = loader;

        const WebStorageAdapter = Java.use(clazz);
        WebStorageAdapter.deleteAllData.implementation = function () {
          console.log('WebStorage.deleteAllData() called');
          return this.deleteAllData();
        };
        return;
      }
    } catch (e) {
    }

    setTimeout(waitForMethods, 500);
  }

  console.log('Waiting for chromium to be loaded');
  waitForMethods();
});

```